package.json

```
@@ -11,19 +11,22 @@
     "dev:server": "cd pyflask && python -m flask run --port 4242",
     "build": "npm run build:app",
     "build:app": "electron-vite build",
+    "build:mac": "npm run build && npm run build:python-unix && npm run build-mac",
+    "build:win": "npm run build && npm run build:python && npm run build-win",
+    "build:python": "python -m PyInstaller --onedir --clean --noconfirm ./pyflask/app.py --distpath ./out/python --add-data ./paths.config.json;. --collect-all pynwb --collect-all hdmf --collect-all ci_info",
+    "build:python-unix": "python -m PyInstaller --onedir --clean --noconfirm ./pyflask/app.py --distpath ./out/python --add-data ./paths.config.json:. --collect-all pynwb --collect-all hdmf --collect-all ci_info",
```

```suggestion
    "build:mac": "electron-vite build && npm run build:flask && npm run build-mac",
    "build:win": "electron-vite build && npm run build:flask && npm run build-win",
    "build:flask": "python -m PyInstaller --onedir --clean --noconfirm ./pyflask/app.py --distpath ./out/python --add-data ./paths.config.json:. --collect-all pynwb --collect-all hdmf --collect-all ci_info",
```

Perhaps I'm having a "where's Waldo" moment, but the two `build:python` look the same to me

Also suggested rename to be more explicit about this subcommand - while Python is certainly a part of what is built, the full Flask server distribution is much more than just the base Python

Also removing excess subcommands that can be included in the main `build:<platform>` commands to reduce abstraction (let me know if the original structure is actually somehow needed)

---

Windows and Unix-like systems have different separators for the `build:python` command so there do need to be two versions.